gazelle/python/target.go

```diff
@@ -15,12 +15,11 @@
 package python
 
 import (
-	"path/filepath"
-
 	"github.com/bazelbuild/bazel-gazelle/config"
 	"github.com/bazelbuild/bazel-gazelle/rule"
 	"github.com/emirpasic/gods/sets/treeset"
 	godsutils "github.com/emirpasic/gods/utils"
+	"path/filepath"
```

Comment:
Unnecessary and wrong change, we keep standard lib imports at the top.

Reply:
That was changed by my GoLand and I didn't notice.